BoxContentSDK/BoxContentSDK/Requests/BOXFolderItemsRequest.m

```diff
@@ -24,6 +24,8 @@ @interface BOXFolderItemsRequest ()
 
 @implementation BOXFolderItemsRequest
 
+@synthesize rangeStep = _rangeStep;
+
 - (instancetype)initWithFolderID:(NSString *)folderID
 {
     if (self = [super init]) {
```

Comment:
Should probably initialize range step to 0

Reply:
done